app/scripts/modules/core/serverGroup/serverGroup.write.service.js

```diff
@@ -60,6 +60,23 @@ module.exports = angular
       });
     }
 
+    function rollbackServerGroup(serverGroup, application, params={}) {
+      params.asgName = serverGroup.name;
+      params.serverGroupName = serverGroup.name;
+      params.type = 'rollbackServerGroup';
+      params.regions = [serverGroup.region];
+      params.region = serverGroup.region;
```

Comment:
`orca` is just passing almost directly through to cloud driver hence following the same convention here.

Just looking at CD to see if we can't tidy up some of this.